tests/test_bdc_pipeline.py

```diff
@@ -13,6 +13,7 @@
 
 class DummyStepOne(Step):
```

Comment:
Maybe add a testcase for checking the exisitng columns?

Reply:
Done